src/app/rundown-view/components/minishelf/mini-shelf.component.scss

```diff
@@ -0,0 +1,42 @@
+.c-mini-shelf {
+  min-width: 270px;
+  max-width: min-content;
+  padding: 0;
+  margin: 5px 0 5px 20px;
+  border-radius: 5px;
+  overflow: hidden;
+  border: solid 1px #eee;
+  background-color: blue;
+  box-shadow: 0 2px 2px 2px rgba(0,0,0,0.2);
+
+  &__thumbnail {
+    width: 100%;
+    height: 60px;
+    object-fit: cover;
+    object-position: 50% 60%;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  &__note {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 18px;
+    background: #1768ff;
+    padding: 2px;
+    text-shadow: black 0 0 2px;
+    font-weight: 450;
+    font-size: 12px;
+  }
+
+  &__title {
+    color: white;
+  }
+
+  &__duration {
+    color: yellow;
```

Comment:
Personal opinion incoming: I don't see why this should be yellow, instead of white as the rest of the text.